README.md

```diff
@@ -20,17 +20,29 @@ This platform has been developed at the [RVMI lab](http://www.rvmi.aau.dk/).
 This project has received funding from the European Union’s Horizon 2020
 research and innovation programme under grant agreement no. 723658, [Scalable4.0](https://www.scalable40.eu/).
 
+## Compatibility
+
+SkiROS is compatible with Ubuntu 16.04/ROS Kinetic, Ubuntu 18.04/ROS Melodic and Ubuntu 20.04/ROS Noetic, python 2 and 3.
+
 ## Install
 
 To use SkiROS you must have [ROS](https://wiki.ros.org/ROS/Installation) installed on your machine.  
+You also need [pip](https://pip.pypa.io/en/stable/installing/) to install python dependencies.
 
-You must include this package in your catkin workspace and compile with catkin. Also, it is highly recommended to include in the catkin workspace the [base skill set](https://github.com/RVMI/skiros2_std_lib).
+Install xterm
```

Comment:
Xterm dependency is already specified in rosdep

Reply:
Oh I never ran rosdep, maybe it's useful to add a 'rosdep install' in the readme as well then? Not sure about the arguments you'd pass it though.

Reply:
Good point. Not sure if I can modify your pull request, the commands for rosdep is this:
> rosdep install --from-paths . --ignore-src --rosdistro=$ROS_DISTRO -y
Where $ROS_DISTRO should be defined or replaced with your ROS distro (e.g. melodic).

Reply:
Should be possible somehow, but I'm not very familiar with the whole pull request system either.
Solved in https://github.com/RVMI/skiros2/pull/40/commits/eaa4c1c68f66cba602eb2ff35b20c9911717be9c